### ISSUE TITLE
OpenGovLD export

### ISSUE THREAD

**Issue by akuckartz** [okfde/offenesparlament.de#39]
I am prepared to add support for OpenGovLD (https://github.com/OpenGovLD/specs) export around summer 2015.

**Comment by akuckartz** [comment]
This page contains information on available data formats:
http://offenesparlament.de/info/daten

**Comment by pudo** [comment]
Jo, ich würde OParl nehmen, wg. weiterer Verbreitung (echter Datenstandard).

**Comment by akuckartz** [comment]
OParl ist zwar eine Spezifikation, von einem "Standard" ist diese jedoch leider sehr weit entfernt (das Adjektiv "echt" ändert daran nichts). Selbst die zurückhaltenden Kriterien von Open Stand (https://open-stand.org/about-us/principles/) werden von OParl überwiegend nicht erfüllt. Das betrifft insbesondere "Due process", "Broad consensus", "Transparency", "Balance" und "Openness", aber auch "technical merit", "global interoperability" und "contribute to the creation of global communities".

Die Gegenüberstellung auf https://github.com/OpenGovLD/specs/wiki/OpenGovLD-vs.-OParl würde ich heute möglicherweise stellenweise etwas weniger scharf formulieren, in der Sache jedoch nicht.

Popolo (eine wesentliche Grundlage von OpenGovLD) ist weltweit bereits seit Jahren verbreiteter als OParl (und wesentliche Teile von OParl hatte ich von Popolo übernommen und in OParl integriert). Noch nicht einmal eine gemeinsame Kampagne z.B. von Vitako, Fraunhofer FOKUS, OKF Deutschland e.V. und deutschen RIS-Herstellern für diese deutsche Speziallösung würde an der weltweiten Entwicklung etwas ändern - insbesondere nicht in positiver Richtung.

Last but not least: OParl spezifiziert kein Exportformat und ist schon deshalb keine Alternative. Siehe https://github.com/OParl/specs/issues/64
